src/AppBundle/Controller/RepositoryController.php

```diff
@@ -40,13 +41,49 @@ public function indexAction(Request $request, Repository $repository)
             }
         }
 
+        $user = $this->getUser();
+        $repositoryStar = $this->get('doctrine')->getRepository('AppBundle:RepositoryStar')->findOneBy([
```

Comment:
PM: could also be moved in `RepositoryEntity` (i.e. in a method `isStarredByUser(User $user)`)